Formula/dashing.rb

```diff
@@ -0,0 +1,27 @@
+class Dashing < Formula
+  desc "Generate Dash documentation from HTML files"
+  homepage "https://github.com/technosophos/dashing"
+  url "https://github.com/technosophos/dashing/archive/0.3.0.tar.gz"
+  sha256 "f6569f3df80c964c0482e7adc1450ea44532d8da887091d099ce42a908fc8136"
+  head "https://github.com/technosophos/dashing.git"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    (buildpath/"src/github.com/technosophos/dashing").install buildpath.children
+    cd "src/github.com/technosophos/dashing" do
+      system "glide", "install"
+      system "make", "build"
```

Comment:
From the Makefile
```

VERSION := $(shell git describe --tags)

build:
	go build -o dashing -ldflags "-X main.version=${VERSION}" dashing.go
```
That won't work unless we use the tag+revision. But I'd prefer we just run `go build` manually and keep using the tarball